src/steamship_langchain/llms/openai.py

```diff
@@ -188,3 +189,102 @@ def _batch(
             pass
 
         return generations, token_usage
+
+
+class OpenAIChat(BaseOpenAIChat):
+    _llm_plugin: PluginInstance
+
+    class Config:
+        """Configuration for this pydantic object."""
+
+        extra = Extra.allow
+
+    def __init__(
+        self, client: Steamship, model_name: str = "gpt-4", moderate_output: bool = True, **kwargs
+    ):
+        super().__init__(client=client, model_name=model_name, **kwargs)
+        plugin_config = {"model": self.model_name, "moderate_output": moderate_output}
+        if self.openai_api_key:
```

Comment:
Just double checking that Pydantic won't accidentally dump `openai_api_key` to logs if someone calls `print` on it